DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateProgress/EditCompleteByDate.cshtml

```diff
@@ -7,15 +7,12 @@
 @{
   var errorHasOccurred = !ViewData.ModelState.IsValid;
   ViewData["Title"] = errorHasOccurred ? "Error: Edit complete by date" : "Edit complete by date";
-  var routeParamsForBackLink = new Dictionary<string, string?>();
+  var routeParamsForBackLink = new Dictionary<string, string?> {
+    { "accessedVia", Model.AccessedVia.ToString() },
+    { "progressId", Model.ProgressId.ToString() },
+    { "returnPage", Model.ReturnPage.ToString() },

```

Comment:
You're going to need to resolve merge conflicts and sort out the return page stuff on these pages now that I've changed how all that works